bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseEnums.java

```diff
@@ -51,39 +56,73 @@ public static AirbaseMode fromValue(int value) {
 
             LOGGER.debug("Unexpected Mode value of \"{}\"", value);
 
-            // Default to auto
-            return AUTO;
+            // Default to cold
```

Comment:
Is there a special reason why you changed the default from AUTO to COLD? In general I think that these defaults are wrong. An `IllegalArgumentException` should be thrown (also applies below). The logger can be removed then, too.

Reply:
Some aircon controllers do not actually support AUTO mode. Indeed, COLD is an arbitrary default. Throwing an exception will render the binding non-operational for models with new/unsupported modes and set to that mode. Should that be done? By providing a default, the binding can still be used within the supported modes.